src/Table/Table.story.tsx

```diff
@@ -374,5 +374,5 @@ export const WithOnHoverActions = () => {
 };
 
 WithOnHoverActions.story = {
-  name: "with hover actions",
+  name: "with on hover actions",
```

Comment:
I think this could be removed all together. By default storybook takes the name of the component `WithOnHoverActions` and transforms it to a sentence case: `With on hover actions`. So we don't have to manually do this for this case.

Reply:
You are correct!